README.md

```diff
@@ -11,8 +11,6 @@ The elevator pitch description of the `RL` is:
 
 Read the project rationale [here](./docs/where_we_are_going.md)
```

Comment:
something went wrong with rebasing? seems this file is stuck to a earlier version of it